client/fixtures/consents.ts

```diff
@@ -1,6 +1,6 @@
 /*
- * Consent for marketing communications for signed in user
- * /idapi/user/consents (proxied to IDAPI /consents?filter=all)
+ * List of consents available to a user (does not require authentication)
+ * /idapi/consents (proxied to IDAPI /consents?filter=all)
```

Comment:
did this change at some point or was this a misunderstanding from the start ?
or option c) we don't know

Reply:
A misunderstanding from the start, I suspect, but I'm not sure.